### PR TITLE
Integrate react router. Prepared a few raw components to be styled.

### PR DESCRIPTION
**Pull request by alienalien13**

### CODE REVIEW

File: src/components/Search/SearchInput/SearchInput.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  componentDidMount() {
+    // console.log('SearchInput mounted')
+  }
+
+  render() {
+    // const {  } = this.props
+
+    return (
+      <div>
+        SearchInput
```
Comment (maybelinot) [inline src/components/Search/SearchInput/SearchInput.js:15]: How does this work?

File: src/components/Search/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+import { app, header } from './index.module.css'
+
+import ClearButton from './ClearButton/ClearButton'
+import SearchButton from './SearchButton/SearchButton'
+import SearchInput from './SearchInput/SearchInput'
```
Comment (maybelinot) [inline src/components/Search/index.js:7]: maybe **SearchInputField** or **SearchField**?

File: src/components/Search/ClearButton/ClearButton.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+
+// import {  } from './index.module.css'
+
+class ClearButton extends Component<*, *> {
```
Comment (flashlee) [inline src/components/Search/ClearButton/ClearButton.js:5]: What does `<*, *>` mean?

File: src/components/Search/SearchInput/SearchInput.js (hunk 1 above)
Comment (alienalien13) [reply src/components/Search/SearchInput/SearchInput.js:15]: This is just a text which will be replaced with some content. @maybelinot

File: src/components/Search/index.js (hunk 2 above)
Comment (alienalien13) [reply src/components/Search/index.js:7]: Could be, but what is wrong with SearchInput? 🙂

File: src/components/Search/ClearButton/ClearButton.js (hunk 3 above)
Comment (alienalien13) [reply src/components/Search/ClearButton/ClearButton.js:5]: @flashlee "any props" and "any state"
This is related to types, so we should setup types system as well for better data flow.
Eslint forces to define it at least somehow, so now it just "something".

File: src/components/Search/index.js (hunk 2 above)
Comment (maybelinot) [reply src/components/Search/index.js:7]: Never met this before, sounded weird to me, but looks like it's used - let's keep, sorry for the noise :D

**Review by maybelinot — APPROVED** [review]
LGTM

**Pull request merged**